jsonnet/kube-prometheus/components/prometheus-adapter.libsonnet

```diff
@@ -39,7 +39,7 @@ local defaults = {
       },
       memory: {
         containerQuery: 'sum(container_memory_working_set_bytes{<<.LabelMatchers>>,container!="",pod!=""}) by (<<.GroupBy>>)',
-        nodeQuery: 'sum(node_memory_MemTotal_bytes{job="node-exporter",<<.LabelMatchers>>} - node_memory_MemAvailable_bytes{job="node-exporter",<<.LabelMatchers>>}) by (<<.GroupBy>>)',
+        nodeQuery: 'sum(node_memory_MemTotal_bytes{job="node-exporter",<<.LabelMatchers>>} - node_memory_MemAvailable_bytes{job="node-exporter",<<.LabelMatchers>>}) by (<<.GroupBy>>) or sum(windows_cs_physical_memory_bytes - windows_memory_available_bytes) by (<<.GroupBy>>)',
```

Comment:
Same as above, I don't think the windows metrics contain an `instance` label. We should correlate the `windows-exporter` pod value with the node on which it is running.

Reply:
@dgrisonnet do you mean to say we should be adding a job="windows-exporter" label in this query?